lib/school/app.rb

```diff
@@ -2,14 +2,29 @@
 require_relative 'student'
 require_relative 'teacher'
 require_relative 'rental'
+require 'json'
+# rubocop:disable Metrics
 
 class App
   attr_accessor :people, :rentals, :books
 
   def initialize
-    @rentals = []
     @books = []
     @people = []
+    @rentals = []
+    @books_path = './../file/books.json'
+    @people_path = './../file/people.json'
+    @rentals_path = './../file/rentals.json'
+    initialize_files
+  end
```

Comment:
- [ ] The app is not running properly, and the access path to your files is not correct. Let's fix that

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/40503704/181021287-1ce03993-78ea-404b-86c8-615f83a3e1aa.png">